packages/grid/data-grid/src/tests/keyboard.DataGrid.test.tsx

```diff
@@ -171,6 +171,14 @@ describe('<DataGrid /> - Keyboard', () => {
     expect(getActiveCell()).to.equal('4-0');
   });
 
+  it('Space only should go to the bottom of the page even with small number of rows', () => {
+    render(<KeyboardTest nbRows={4} />);
+    getCell(0, 0).focus();
+    expect(getActiveCell()).to.equal('0-0');
+    fireEvent.keyDown(document.activeElement!, SPACE_KEY);
```

Comment:
I personally think it would be much better with:

```suggestion
    fireEvent.keyDown(document.activeElement!, { key: ' ' });
```

It optimizes for duplication of information and explicitlness. You will find 18 matches in the core with `, { key: ' ' }`.

Reply:
Also, it's weird that we have `code` (unused) or `shiftKey` (default) defined in:

https://github.com/mui-org/material-ui-x/blob/76948930e3a9fa6d0d03fe60c54b60d3a60e2202/packages/grid/data-grid/src/tests/keyboard.DataGrid.test.tsx#L20

Reply:
When you fire a keydown event you have code and shiftKey, we have different behavior for space+shift.
Also if you start using code in your implementation, your test won't work anymore.

Reply:
This was my point, code shouldn't be used, shift is already the default.